lms/templates/courseware/course_about.html

```diff
@@ -269,12 +270,48 @@ <h2 class="title">
               </li>
               % endif
 
+            ## Effort
             % if get_course_about_section(request, course, "effort"):
-              <li class="important-dates-item"><span class="icon fa fa-pencil" aria-hidden="true"></span><p class="important-dates-item-title">${_("Estimated Effort")}</p><span class="important-dates-item-text effort">${get_course_about_section(request, course, "effort")}</span></li>
+              <li class="important-dates-item">
+                <span class="icon fa fa-pencil" aria-hidden="true"></span>
+                <p class="important-dates-item-title">${_("Estimated Effort")}</p>
+                <span class="important-dates-item-text effort">${get_course_about_section(request, course, "effort")}</span>
+              </li>
+            % endif
+
+            ## Organization / Institute
+            % if course.display_org_with_default:
+              <li class="important-dates-item">
+                <span class="icon fa fa-university" aria-hidden="true"></span>
+                <p class="important-dates-item-title">${_("Institution")}</p>
+                <span class="important-dates-item-text institution">${course.display_org_with_default}</span>
+              </li>
             % endif
 
-            ##<li class="important-dates-item"><span class="icon fa fa-clock-o" aria-hidden="true"></span><p class="important-dates-item-title">${_('Course Length')}</p><span class="important-dates-item-text course-length">${_('{number} weeks').format(number=15)}</span></li>
+            ## Language
+            % if course_details.language:
+              <li class="important-dates-item">
+                <span class="icon fa fa-language" aria-hidden="true"></span>
+                <p class="important-dates-item-title">${_("Language")}</p>
+                <span class="important-dates-item-text language">${get_language_info(course_details.language)['name_local']}</span>
+              </li>
+            % endif
 
+            ## Pull course run info from course discovery
+            <%
+              course_run_details = get_course_run_details(course.id.to_deprecated_string(), ['weeks_to_complete'])
+            %>
+
+            ## Duration
+            % if course_run_details:
```

Comment:
Because you're passing fields to `get_course_run_details` above, you'll always have a `weeks_to_complete` key, meaning that this object will never be falsey - on my devstack, emptying the Weeks to Complete field results in the blurb "None Weeks" on this line.

Reply:
Ah, good point. I'll just check if any info is associated with the key, then.